src/Classes/TradeQuery.lua

```diff
@@ -814,7 +494,8 @@ function TradeQueryClass:PriceItemRowDisplay(controls, str_cnt, slotTbl, top_pan
 		elseif controls['uri'..str_cnt].buf == "" then
 			activeSlotRef.pbURL = ""
 		end
-		return validURL and self:PriceBuilderCanSearch(controls) and self:PriceBuilderCanFetch(controls)
+		local isSearching = controls['priceButton'..str_cnt].label == "Searching..."
```

Comment:
This feels kind of hokey to me.  Might see if there's a better way to do this logic when I dig in some more.

Reply:
I didn't touch any UI stuff except a few lines and this was one of them. I had to throw in an ugly fix while testing the rate limiter. This is one of the reasons why I think a UI review is necessary before merging.